File: app/src/main/java/com/unpas/masteringmaths/student/activity/StudentKonsepActivity.java
package com.unpas.masteringmaths.student.activity;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.webkit.WebView;

import com.unpas.masteringmaths.R;

import static com.unpas.masteringmaths.utils.UtilsConstant.COMPETENT_NUMBER;

public class StudentKonsepActivity extends AppCompatActivity {

    private WebView konsep;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_student_konsep);

        konsep = findViewById(R.id.main_konsep);
        konsep.getSettings().setBuiltInZoomControls(true);

        int competentNumber = getIntent().getIntExtra(COMPETENT_NUMBER, 0);

        switch (competentNumber){
            case 0:
                konsep.loadUrl("file:///android_asset/petakonseprelasi.html");
                break;
            case 1:
                konsep.loadUrl("file:///android_asset/petakonseppythagoras.html");
                break;
            case 2:
                konsep.loadUrl("file:///android_asset/petakonseppolabilangan.html");
                break;
            case 3:
                konsep.loadUrl("file:///android_asset/petakonsepgarislurus.html");
                break;
            case 4:
                konsep.loadUrl("file:///android_asset/petakonsepkartesius.html");
                break;
            case 5:
                konsep.loadUrl("file:///android_asset/petakonsepspldv.html");
                break;
        }
    }
}

File: app/src/main/java/com/unpas/masteringmaths/student/activity/StudentKonsepActivity.kt
package com.unpas.masteringmaths.student.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.unpas.masteringmaths.R
import com.unpas.masteringmaths.utils.UtilsConstant
import kotlinx.android.synthetic.main.activity_student_kompetesi.*
import kotlinx.android.synthetic.main.toolbar_layout.*

class StudentKonsepActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_student_kompetesi)
        setSupportActionBar(toolbar)
        supportActionBar?.title = "Peta Konsep"

        main_content.settings.builtInZoomControls = true
        val competentNumber =
            intent.getIntExtra(UtilsConstant.COMPETENT_NUMBER, 0)

        when (competentNumber) {
            0 -> main_content.loadUrl("file:///android_asset/petakonseprelasi.html")
            1 -> main_content.loadUrl("file:///android_asset/petakonseppythagoras.html")
            2 -> main_content.loadUrl("file:///android_asset/petakonseppolabilangan.html")
            3 -> main_content.loadUrl("file:///android_asset/petakonsepgarislurus.html")
            4 -> main_content.loadUrl("file:///android_asset/petakonsepkartesius.html")
            5 -> main_content.loadUrl("file:///android_asset/petakonsepspldv.html")
        }
    }
}